README.md

```diff
@@ -0,0 +1,62 @@
+# Telerik Reporting Virtual Classroom Training Starter Project
+
+Welcome to the sample project repository of the
+[Telerik Reporting getting started video course](https://learn.telerik.com/learn/course/internal/view/elearning/38/telerik-reporting).
+This training course is developed to help you get started with the Telerik Reporting set of services and features.
+It aims to put you in the shoes of a report designer and an engineer that is integrating a reporting solution into an existing application.
+
+In this repo, you will find the starter source code needed for the hands-on exercises that are part of the different chapters throughout the training.
+You will also find two final states of the exercises in branches of this repository:
+* [basic-complete](https://github.com/telerik/RpsTrackerBlazorReportingStart/tree/basics-complete) contains the state of the exercises when you complete chapter 3 
+* [advanced-complete](https://github.com/telerik/RpsTrackerBlazorReportingStart/tree/advanced-complete) contains the state of the exercises
+when you complete the next chapters
+
+## About the Telerik Reporting Video Course
+
+All users with an active trial or commercial license have access to the 
+[Telerik Reporting video course](https://learn.telerik.com/learn/course/internal/view/elearning/38/telerik-reporting)
+available in [Telerik's Virtual Classroom](https://learn.telerik.com/learn).
+
+After the Welcome chapter, the second chapter makes an overview of what tools the 
+[Telerik Reporting](https://www.telerik.com/products/reporting?utm_medium=referral&utm_source=github&utm_campaign=reporting-awareness-gh-public-readme)
+product includes and what a report definition is.
+
+Chapter 3 guides you in the process of designing a basic report using the
+[Standalone Report Designer](https://docs.telerik.com/reporting/designing-reports/report-designer-tools/desktop-designers/standalone-report-designer/overview)
+and integrating it into a sample Blazor application.
```

Comment:
... into a sample Blazor application by using the Blazor Report Viewer ?

Reply:
Good idea. Added.